packages/library-base/src/commonMain/kotlin/io/realm/internal/platform/CoroutineUtils.kt

```diff
@@ -8,13 +8,13 @@ import kotlin.coroutines.EmptyCoroutineContext
 /**
  * Returns a default Realm write dispatcher for a Realm opened on the calling thread.
  */
-expect fun singleThreadDispatcher(id: String): CoroutineDispatcher
+public expect fun singleThreadDispatcher(id: String): CoroutineDispatcher
```

Comment:
Why are these public? Just due to test usage?

Reply:
Yes, they are used in enough places that using suppress everywhere seems wrong.